src/Database/Schema/PostgresSchema.php

```diff
@@ -468,7 +467,7 @@ protected function _keySql($prefix, $data)
                 ' FOREIGN KEY (%s) REFERENCES %s (%s) ON UPDATE %s ON DELETE %s DEFERRABLE INITIALLY IMMEDIATE',
                 implode(', ', $columns),
                 $this->_driver->quoteIdentifier($data['references'][0]),
-                $this->_driver->quoteIdentifier($data['references'][1]),
+                $this->_convertFkColumnsReference($data['references'][1]),
```

Comment:
I would avoid abbreviations in the method names. You could call this `_convertConstraintColumns()` instead.